Calculator/Layout.java

```diff
@@ -0,0 +1,63 @@
+import java.awt.Component;
+
+/**
+ * determine the layout
+ * @author Udit Saxena
+ *
+ */
+public class Layout {
+
+	private int screenSize[][];
+
+	/**
+	 * Adds the component comp to the layout, associating it with the string
+	 * specified by name
+	 * 
+	 * @param comp
+	 *            - Component Class object
+	 */
+	void addLayoutComponent(Component comp) {
+	}
+
+	/**
+	 * Calculates the minimum size dimensions, given the components it contain
+	 */
+	void minimumLayoutSize() {
```

Comment:
Method name should specify whether it is used to set or get. This can be setMinimumLayoutSize. and also some size needs to be passed in this method as parameters.